.changeset/spotty-chairs-pretend.md

```diff
@@ -0,0 +1,8 @@
+---
+"@opennextjs/aws": patch
```

Comment:
Maybe this should be minor (new feature) instead of patch (bug fix) ?

Reply:
I think it's ok this way, technically there is no new feature here, and it should have no impact at all for end user (even people with custom overrides)
Other PR that will depend on this one will be minor like #665